model/feedback/feedback.go

```diff
@@ -0,0 +1,30 @@
+package feedback
```

Comment:
can we just put one offs like this in the model package? there's no real reason to have its own package here and the overhead is non-zero.

Reply:
moved to the model package